lib/Doctrine/Migrations/Version.php

```diff
@@ -221,7 +226,7 @@ private function getWriteSqlFileMigratorConfig() : MigratorConfig
     private function getExecutedAtDatabaseValue() : string
     {
         return Type::getType(MigrationTable::MIGRATION_EXECUTED_AT_COLUMN_TYPE)->convertToDatabaseValue(
-            new DateTimeImmutable(),
+            (new DateTimeImmutable('now', new DateTimeZone(date_default_timezone_get())))->setTimezone(new DateTimeZone('UTC')),
```

Comment:
You can drop `new DateTimeZone(date_default_timezone_get()))`, it's default behavior.